runtime-modules/content/src/tests/nft/update_nft_limit.rs

```diff
@@ -165,3 +166,30 @@ fn default_channel_nft_limits_set_successfully() {
         );
     })
 }
+
+#[test]
+fn toggle_nft_limits_ok_with_event_deposited_and_status_changed() {
```

Comment:
Probably worth adding at least one invalid origin case (for example: lead)

Reply:
added in e84b6b9d6f9d9021c4fa7ff6d58700270c65ad6e